package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "mike-proxy",
+  "version": "1.0.0",
+  "description": "Proxy server for fec3-galadriel",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start:dev": "nodemon server/server.js",
```

Comment:
For future reference, you can just point it at your server directory and it'll run index.js automatically afaik.